### ISSUE TITLE
How to fill value in edit form

### ISSUE THREAD

**Issue by eshaiju** [jim-moody/react-phone-input-auto-format#2]
How can we add value from the state into this field on load? Following is not working

```
<PhoneInput
     name='cellPhone'
     value={this.state.cellPhone}
      onChange={onCellPhoneChange}
/>
```

**Comment by kristajg** [comment]
Added PR https://github.com/jim-moody/react-phone-input-auto-format/pull/4

**Comment by jim-moody** [comment]
Fixed by PR #3
